analyses/org.osate.analysis.model/src/org/osate/model/analysis/traversal/AnalysisModelTraversal.java

```diff
@@ -0,0 +1,147 @@
+package org.osate.model.analysis.traversal;
```

Comment:
Switch model and analysis in package name to match the naming pattern in this plugin.
Also, maybe use `util` instead of `traversal` to avoid anothe package with a single class (...util exists in src-gen).

Reply:
I was purposely trying to separate this class from the generated code.

Reply:
It's already in a different source directory. I find it confusing to have packages org.osate.model.analysis and org.osate.analysis model.

Reply:
Renamed the ecore packages to make this work.  Changed org.osate.model.analysis.traversal to org.osate.model.analysis.util